fabric/build.gradle

```diff
@@ -5,18 +5,20 @@ buildscript {
         }
 
         /*
-        This implements the forge eventbus transformers, through the 2 following ways
+        This implements transformers to apply net.minecraftforge.eventbus.EventSubclassTransformer
+        through the 2 following ways
 
         Architectury fabric transformer
-        - Transforms relevant classes at runtime
-        - Reason for dependencies implementation
-        - class EventBusTransform in repo
+        - Transforms relevant classes in dev environment at runtime
+        - Reason for the configuration runtimeOnly
+        - class EventBusTransform in dependency
 
         GradleTask applyEventBus
-        - Transforms relevant classes after shadowing jar
-        - Reason for buildscript classpath
+        - Transforms relevant classes after shadowing jar for use in final jar
+        - Reason for being in buildscript classpath
```

Comment:
It's in the buildscript classpath so that it can be added to the list of architectury transformers for runtime.
The applyEventBus task uses the runtime classpath since eventbus need's to be able to load super classes using the jvm class loader.

Reply:
Because the event bus tries to load super classes from the jvm in the EventSubclassTransformer